modules/sandboxed-containers-deleting-kataconfig-web-console.adoc

```diff
@@ -6,7 +6,18 @@
 [id="sandboxed-containers-deleting-kataconfig-web-console_{context}"]
 = Deleting the KataConfig custom resource using the web console
 
-Deleting the `KataConfig` custom resource (CR) removes and uninstalls the `kata` runtime and its related resources from your cluster. The deletion typically takes between ten to forty minutes, depending on the size of the deployment.
+Deleting the `KataConfig` custom resource (CR) removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+[IMPORTANT]
+====
+Deleting the `KataConfig` CR automatically reboots the worker nodes. The reboot can take anywhere from ten minutes to over an hour. Factors that impede reboot time are as follows:
+
+* A larger {product-title} deployment with a greater number of worker nodes.
+* Activation of BIOS and Diagnostics.
```

Comment:
See previous comment about this capitalization

Reply:
Changed.